core/server/errors/no-database-found.js

```diff
@@ -0,0 +1,11 @@
+function NoDatabaseFound(message) {
+    this.message = message;
+    this.stack = new Error().stack;
+    this.statusCode = 404;
+    this.errorType = this.name;
+}
+
+NoDatabaseFound.prototype = Object.create(Error.prototype);
+NoDatabaseFound.prototype.name = 'NoDatabaseFound';
+
+module.exports = NoDatabaseFound;
```

Comment:
Do we really need a specific error type for this? An error like this should be a generic 500 error with a clear message IMO?